gateways/node_api_gateway.py

```diff
@@ -86,25 +89,33 @@ def get_address_search(
         return self.hathor_core_client.get(
             ADDRESS_SEARCH_ENDPOINT,
             params=params,
-            timeout=10,
+            timeout=14,  # lambda timeout 15s
```

Comment:
This one is also part of a Lambda that makes other requests besides this.

I think it should remain as 10s.